README.md

```diff
@@ -1,2 +1,26 @@
 # Functional.CQS
-Defines CQS-style contracts and provides AOP facilities over those contracts using [Functional](https://github.com/JohannesMoersch/Functional) primitives
+
+Defines [CQS-style contracts](https://github.com/RyanMarcotte/Functional.CQS/tree/master/src/Functional.CQS) using [Functional primitives](https://github.com/JohannesMoersch/Functional):
```

Comment:
Out of curiosity, can the first link here just be a relative path `(src/Functional.CQS)`, rather than the full URL?

Reply:
The link checker broke when it was a relative path :(